client/components/mma/accountoverview/ProductCard.tsx

```diff
@@ -130,6 +130,7 @@ export const ProductCard = ({
 
 	const showSwitchButton =
 		isEligibleToSwitch &&
+		!hasCancellationPending &&
```

Comment:
unrelated to this PR, but just realised the switch button text is `Change to monthly + extras`, which should probably be dynamic?

Reply:
You're right! 😱 I'll fix that in a separate PR…